homeassistant/components/fitbit/sensor.py

```diff
@@ -765,3 +788,40 @@ def _handle_coordinator_update(self) -> None:
         self.device = self.coordinator.data[self.device.id]
         self._attr_native_value = self.device.battery
         self.async_write_ha_state()
+
+
+class FitbitBatteryLevelSensor(CoordinatorEntity, SensorEntity):
```

Comment:
We're missing the coordinator type in the `CoordinatorEntity` annotation. It's generic.

Reply:
Example:
https://github.com/home-assistant/core/blob/a78e3f7b0f1e1424de84c00297f4d0da7202d0a2/homeassistant/components/wled/models.py#L9

Reply:
```suggestion
class FitbitBatteryLevelSensor(
    CoordinatorEntity[FitbitDeviceCoordinator], SensorEntity
):
```